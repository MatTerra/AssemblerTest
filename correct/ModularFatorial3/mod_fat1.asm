mod_main: begin
mod_fat: extern
n: extern
public aux
section data
aux: space
section text
input n
load n
jmp mod_fat
end
