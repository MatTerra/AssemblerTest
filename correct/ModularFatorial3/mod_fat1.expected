> 5
PC <- 2
ACC <- 0

PC <- 4
ACC <- 5

PC <- 7
ACC <- 5

PC <- 9
ACC <- 4

PC <- 11
ACC <- 4

PC <- 13
ACC <- 4

PC <- 15
ACC <- 20

PC <- 17
ACC <- 20

PC <- 19
ACC <- 4

PC <- 22
ACC <- 4

PC <- 24
ACC <- 4
OUTPUT: 20

PC <- 25
ACC <- 4

